src/Provider/PointCloudProvider.js

```diff
@@ -206,6 +219,8 @@ export default {
             // eslint-disable-next-line no-console
             console.log('LAYER metadata:', root);
             layer.root = root;
+            root.findChildrenByName = findChildrenByName.bind(root, root);
```

Comment:
why you use `bind`?

Reply:
To do currying and pre-set the first argument to `root`.